app/scripts/models/reliers/relier.js

```diff
@@ -75,6 +75,7 @@ define([
           self.importSearchParam('setting');
           self.importSearchParam('entrypoint');
           self.importSearchParam('campaign');
+          self.importSearchParam('sync_chooser');
```

Comment:
Is this left over from a previous, search-param-based version?  AFAICT it's not referenced elsewhere.